dynamiq/nodes/agents/react.py

```diff
@@ -948,33 +948,25 @@ def _run_agent(
                     if not original_streaming_enabled:
                         self.llm.streaming.enabled = True
 
-                    # Add the callback to the config
-                    if config.callbacks is None:
-                        config.callbacks = []
-                    config.callbacks.append(streaming_callback)
+                    # Create a config for the LLM that only has the streaming callback
+                    llm_config = config.model_copy(deep=False)
+                    llm_config.callbacks = [streaming_callback]
```

Comment:
Why tracing callback or some other user custom callbacks should be removed?
For duplication fix it seems needs to remove particular callback, not override all